app/src/main/java/com/rage/clamber/Adapters/CommentsRecyclerViewAdapter.java

```diff
@@ -48,6 +48,8 @@ public void onBindViewHolder(CommentsViewHolder holder, int position) {
         Comment oneComment = comments.get(position);
         holder.commentsTextView.setText(oneComment.getComment());
 
+        //Gets the date in milliseconds and creates a DateTime object from it. Then uses DateUtils
```

Comment:
Should specify UTC, I know it's nitpicky, but timezones and time are finicky, best to be explicit